source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -732,6 +732,54 @@ def _tag_by_mode(responses):
             _tag_journey_by_mode(j)
 
 
+def tag_robust_journeys(responses):
```

Comment:
Should this private like the other tagging function?
```suggestion
def _tag_robust_journeys(responses):
```

That said, I see that `is_robust_journey` probably can't be private because of the tests... :thinking: